app/utils/trackableEvents.js

```diff
@@ -0,0 +1,18 @@
+export const JOIN_A_WORKSPACE = 'join_a_workspace';
```

Comment:
Create a `log` folder under `app/utils` and move this file and `log.js` as well.